sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -91,6 +92,26 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
             return builder;
         }
 
+        public static IWebJobsBuilder AddServiceBusScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
```

Comment:
Are we expecting customers to call this when configuring their Function apps?   It seems like something that would be more infrastructure-focused.   Should we consider making it `internal` rather than `public`?

Reply:
We are not expecting function apps customers will be calling this, but we need to be public as ScaleController will be calling  this.